src/main/java/io/openliberty/tools/common/plugins/config/ServerConfigDocument.java

```diff
@@ -238,6 +216,95 @@ private void initializeAppsLocation(CommonLoggerI log, File serverXML, File conf
         }
     }
 
+
+    /**
+     * jvm.options file read order
+     *   1. {wlp.user.dir}/
+     *   2. {server.config.dir}/configDropins/defaults/
+     *   3. {server.config.dir}/
+     *   4. {server.config.dir}/configDropins/overrides/
+     * @throws FileNotFoundException
+     * @throws Exception
+     */
+    public void processJvmOptions() throws FileNotFoundException, Exception {
+        final String jvmOptionsString = "jvm.options";
+        parsePropertiesFromFile(new File(libertyDirectoryPropertyToFile.get(ServerFeatureUtil.WLP_USER_DIR), jvmOptionsString));
+        parsePropertiesFromFile(getFileFromConfigDirectory(CONFIGDROPINS_DEFAULT + File.separator + jvmOptionsString));
+        parsePropertiesFromFile(getFileFromConfigDirectory(jvmOptionsString));
+        parsePropertiesFromFile(getFileFromConfigDirectory(CONFIGDROPINS_OVERRIDES + File.separator + jvmOptionsString));
+    }
+
+    /**
+     * server.env file read order
+     *   1. {wlp.install.dir}/etc/
+     *   2. {wlp.user.dir}/shared/
+     *   3. {server.config.dir}/
+     * @param serverEnvFile
+     * @throws Exception
+     * @throws FileNotFoundException
+     */
+    public void processServerEnv() throws Exception, FileNotFoundException {
+        final String serverEnvString = "server.env";
+        parsePropertiesFromFile(new File(libertyDirectoryPropertyToFile.get(ServerFeatureUtil.WLP_INSTALL_DIR), "etc" + File.separator + serverEnvString));
+        parsePropertiesFromFile(new File(libertyDirectoryPropertyToFile.get(ServerFeatureUtil.WLP_USER_DIR), "shared" + File.separator + serverEnvString));
+        parsePropertiesFromFile(new File(libertyDirectoryPropertyToFile.get(ServerFeatureUtil.SERVER_CONFIG_DIR), serverEnvString));
```

Comment:
So this is what I suggested for processing jvm.options above. You are loading server.env from SERVER_CONFIG_DIR, not from configDirectory.